### PR TITLE
Ex 4 to 6

### PR DESCRIPTION
**Pull request by julianpoma**

### CODE REVIEW

File: 02-OOP-Inheritance/Ex-4-5-6/split/movie.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import import {EventEmitter} from './ee';
```
Comment (maximilianotulian) [inline 02-OOP-Inheritance/Ex-4-5-6/split/movie.js]: Please use a more descriptive name for the `EventEmitter`

File: 02-OOP-Inheritance/Ex-4-5-6/split/movie.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        if(typeof actors === 'Object')
+        {
+            this.castList.push(actors);
+        } else {
+            this.castList = this.castList.concat(actors);
+        }
+    }
+}
+
+export {Movies}
```
Comment (maximilianotulian) [inline 02-OOP-Inheritance/Ex-4-5-6/split/movie.js]: Be careful about the object you're exporting. This is a common error that could give you a nice headache, `Movies`

File: 02-OOP-Inheritance/Ex-4-5-6/split/main.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import {EventEmitter} from './ee';
+import {Actor} from './actor';
+import {Movie} from './movie';
+import {Logger} from './logger';
+
+let Social = {
+  share: function(friendName)
```
Comment (maximilianotulian) [inline 02-OOP-Inheritance/Ex-4-5-6/split/main.js]: **Minor**: add blank space before `(`

File: 02-OOP-Inheritance/Ex-4-5-6/split/main.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+import {Actor} from './actor';
+import {Movie} from './movie';
+import {Logger} from './logger';
+
+let Social = {
+  share: function(friendName)
+    {
+        console.log(`${friendName} shares this movie: ${this.title}`)
+    },
+  like: function(friendName)
```
Comment (maximilianotulian) [inline 02-OOP-Inheritance/Ex-4-5-6/split/main.js]: I use to create the functions following the next pattern,

```
awesomeFunction: function () {

}
```

Feel free to not do it.

File: 02-OOP-Inheritance/Ex-4-5-6/split/movie.js (hunk 1 above)
Comment (julianpoma) [inline 02-OOP-Inheritance/Ex-4-5-6/split/movie.js]: Ok!

File: 02-OOP-Inheritance/Ex-4-5-6/split/movie.js (hunk 2 above)
Comment (julianpoma) [inline 02-OOP-Inheritance/Ex-4-5-6/split/movie.js]: Ooops fixed!

File: 02-OOP-Inheritance/Ex-4-5-6/split/main.js (hunk 3 above)
Comment (julianpoma) [inline 02-OOP-Inheritance/Ex-4-5-6/split/main.js]: Done!

File: 02-OOP-Inheritance/Ex-4-5-6/split/main.js (hunk 4 above)
Comment (julianpoma) [inline 02-OOP-Inheritance/Ex-4-5-6/split/main.js]: :)

**Pull request merged**